Mjolnir/src/Subsystems/Chassis.cpp

```diff
@@ -19,10 +19,10 @@
 
 Chassis::Chassis() : Subsystem("Chassis") {
     // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
-    right1Wheel = RobotMap::chassisright1Wheel;
-    right2Wheel = RobotMap::chassisright2Wheel;
-    left1Wheel = RobotMap::chassisleft1Wheel;
-    left2Wheel = RobotMap::chassisleft2Wheel;
+	static std::shared_ptr<WPI_TalonSRX> left1Wheel;
```

Comment:
I think these don't need to be `shared_ptr<>`'s. Just 

    static WPI_TalonSRX left1Wheel;

Etc.

Reply:
Also remove `static` so ... 

    WPI_TalonSRX left1Wheel(kLeftWheelID);